### PR TITLE
MT: allow use of SQLite for installation.

### PR DESCRIPTION
**Pull request by parkr**
Fixes #223

To use this patch, put the following in your `Gemfile`:

```ruby
gem "jekyll-import", github: "jekyll/jekyll-import", branch: "mt-use-sequel"
```

Then run `bundle install` and `bundle exec irb -rjekyll-import`. Then run the command per the docs specifying the `dbname` option as the path to your SQLite database file.

### CODE REVIEW

File: lib/jekyll-import/importers/mt.rb [hunk 1]
```diff
@@ -65,27 +67,10 @@ def self.specify_options(c)
...
-        if options['port']
-          db_opts[:port] = options['port']
-        end
-
-        if engine != 'mysql' && engine != 'postgres'
-          abort("unsupported --engine value supplied '#{engine}'. Only 'mysql' and 'postgres' are supported")
-        end 
-
-        db = Sequel.public_send(engine, dbname, db_opts)
+        db = database_from_opts(opts)
```
Comment (sixten) [inline lib/jekyll-import/importers/mt.rb]: Should be `options`.

File: lib/jekyll-import/importers/mt.rb [hunk 2]
```diff
@@ -243,6 +228,34 @@ def self.suffix(entry_type)
           entry_type
         end
       end
+
+      def self.database_from_opts(opts)
+        engine   = options.fetch('engine', 'mysql')
```
Comment (sixten) [inline lib/jekyll-import/importers/mt.rb]: Should be `opts`.

File: lib/jekyll-import/importers/mt.rb [hunk 3]
```diff
@@ -243,6 +228,34 @@ def self.suffix(entry_type)
...
+        dbname   = options.fetch('dbname')
+        user     = options.fetch('user')
+        pass     = options.fetch('password', "")
+        host     = options.fetch('host', "localhost")
+
+        db_connect_opts = {
```
Comment (sixten) [inline lib/jekyll-import/importers/mt.rb]: Probably not much point building this outside of the MySQL/PostgreSQL case. Or fetching the user/password/host options, really.

**Comment by parkr** [comment]
@sixten Would you mind trying again with my patch?

**Comment by parkr** [comment]
@jekyllbot: merge +minor

**Pull request merged**